test/pylib/scylla_cluster.py

```diff
@@ -160,7 +160,8 @@ def make_scylla_conf(mode: str, workdir: pathlib.Path, host_addr: str, seed_addr
 # [--overprovisioned, --smp=1, --abort-on-ebadf], [--smp=2] -> [--overprovisioned, --smp=2, --abort-on-ebadf]
 # [], [--experimental-features, raft, --experimental-features, broadcast-tables] ->
 # [--experimental-features, raft, --experimental-features, broadcast-tables]
-def merge_cmdline_options(base: List[str], override: List[str]) -> List[str]:
+def merge_cmdline_options(
+        base: List[str], override: List[str], appending_options: List[str] = ["--logger-log-level"]) -> List[str]:
```

Comment:
yes because tests quite often use --logger-log-level 

for example test_data_resurrection_after_cleanup, test_raft_snapshot_request, test_enable_compacting_data_for_streaming_and_repair_live_update.

if we provide --logger-log-level from test.py CLI and do "MERGE" logic all these parameters that are set in tests will disappear and the test will fail.

according to https://github.com/scylladb/scylladb/issues/14960#issuecomment-2191328794
"Passing --logger-log-level multiple times (e.g. one is passed from the test, another is passed from extra-scylla-cmdline-options) works" and I tested it as well